synapse/storage/prepare_database.py

```diff
@@ -41,7 +41,7 @@ class UpgradeDatabaseException(PrepareDatabaseException):
     pass
 
 
-def prepare_database(db_conn, database_engine, config):
+def prepare_database(db_conn, database_engine, config, data_stores=["main"]):
```

Comment:
> (do we really need the default value?)

It just saves having to update a bunch of tests with the full list of data stores tbh